### ISSUE TITLE
Change icon for remember notes

### ISSUE THREAD

**Issue by angoca** [kmpoppe/noteSolver#31]
When remembering many notes in a small area, it is difficult to remember which one was remembered. It would be cool to change the red marker icon on the map to something else.

**Comment by ericodb** [comment]
I agree that an implementation to highlight notes that are in the list would be great, but since the plugin has not been updated yet, you can **temporarily** use this code:
[https://wiki.openstreetmap.org/wiki/User:DressyPear4/PintarNotas](https://wiki.openstreetmap.org/wiki/User:DressyPear4/PintarNotas)
with the Scripting plugin.
